pico/pom.xml

```diff
@@ -48,7 +48,7 @@
         <module>api</module>
 <!--        <module>spi</module>-->
         <module>types</module>
-<!--        <module>builder</module>-->
+        <module>builder</module>
 <!--        <module>config</module>-->
```

Comment:
I'm assuming all these other modules will come in a future PR, right?

Reply:
Correct.